### ISSUE TITLE
Own data training loss

### ISSUE THREAD

**Issue by wjlszhang** [JiawangBian/sc_depth_pl#28]
Hello @JiawangBian 

I'm training the model with my own data, and found below issue, all the loss (includeing val_loss) except geometry_loss  looks well as below:
<img width="743" alt="6227f68bf1ee67b77d8eb66e64cda9f" src="https://user-images.githubusercontent.com/101317599/203216109-5fab2a52-e6ab-4a47-8ac0-f484751720c1.png">

I wonder whether I should stop the training around 10K iterations, however afer that  point all the other losses still drop down continually.

would you please give some advice on how this happen?

Thanks so much.

**Comment by JiawangBian** [comment]
it looks good. geometry_loss is ok. It cannot be very low because there exist dynamic objects and occlusions that should be inconsistent.

**Comment by wjlszhang** [comment]
thanks much, close it.
